CHANGELOG.md

```
@@ -1,3 +1,22 @@
+## 1.12.1 - 2025-11-02
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)